core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java

```diff
@@ -131,4 +137,44 @@ public String toString() {
       }
     });
   }
+
+  @Override
+  public GetBlockMasterInfoTResponse getBlockMasterInfo(final GetBlockMasterInfoTOptions options)
+      throws TException {
+    return RpcUtils.call(LOG, new RpcUtils.RpcCallable<GetBlockMasterInfoTResponse>() {
+      @Override
+      public GetBlockMasterInfoTResponse call() throws AlluxioException {
+        BlockMasterInfo info = new alluxio.thrift.BlockMasterInfo();
+        for (BlockMasterInfoField field : options.getFilter() != null ? options.getFilter()
+            : Arrays.asList(BlockMasterInfoField.values())) {
+          switch (field) {
+            case LIVE_WORKER_NUM:
+              info.setLiveWorkerNum(mBlockMaster.getWorkerCount());
+              break;
+            case LOST_WORKER_NUM:
+              info.setLostWorkerNum(mBlockMaster.getLostWorkerCount());
+              break;
+            case CAPACITY_BYTES:
+              info.setCapacityBytes(mBlockMaster.getCapacityBytes());
+              break;
+            case USED_BYTES:
+              info.setUsedBytes(mBlockMaster.getUsedBytes());
+              break;
+            case FREE_BYTES:
+              info.setFreeBytes(mBlockMaster.getCapacityBytes() - mBlockMaster.getUsedBytes());
+              break;
+            case CAPACITY_BYTES_ON_TIERS:
+              info.setCapacityBytesOnTiers(mBlockMaster.getTotalBytesOnTiers());
+              break;
+            case USED_BYTES_ON_TIERS:
+              info.setUsedBytesOnTiers(mBlockMaster.getUsedBytesOnTiers());
+              break;
+            default:
+              LOG.warn("Unrecognized master info field: " + field);
```

Comment:
"Unrecognized master info" -> "Unrecognized block master info"

Reply:
Done, thanks

Reply:
I wanted this to be different from the other log message. So, this one should be "block master info", and the other one can be "meta master info" or something different.

Reply:
Sorry for that, I marked it as done when I changed my local code. I will mark as done when I push to my PR in the future. Thanks!